corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -141,6 +145,6 @@ def handle(self, *args, **kwargs):
         print(query_str)
         print(f"BASELINE: {baseline_amt} cases")
         print(f"MATCHED: {hits} cases")
-        print(raw_took_str)
-        print(f"MIN RUNTIME: {min_value} | MAX RUNTIME: {max_value} | AVG RUNTIME: {avg_value}")
+        print(f"MIN TOOK: {min_took} ms | MAX TOOK: {max_took} ms | AVG TOOK: {avg_took} ms")
+        print(f"MIN RUNTIME: {(min_value * 1000):.4g} ms | MAX RUNTIME: {(max_value * 1000):.4g} ms | AVG RUNTIME: {(avg_value * 1000):.4g} ms")
```

Comment:
it could be worth writing this out as a table to make it easier to read. We have some utilities for doing that:
* corehq.util.markup.SimpleTableWriter
* corehq.util.markup.TableRowFormatter and corehq.util.markup.CSVRowFormatter

Here's an example of their usage:
https://github.com/dimagi/commcare-hq/blob/9b62ee75e056ffe6395c0b168437c046c694aabc/corehq/apps/change_feed/management/commands/pillow_topic_assignments.py#L42-L54